src/components/CounterBadge/CounterBadge.visual.st.css

```diff
@@ -0,0 +1,24 @@
+:import {
+  -st-from: "./CounterBadge.st.css";
+  -st-named: overrideStyleParams;
+}
+
+.palette {
+  -st-mixin: CounterBadge(
+    CounterBadgeBgColor "color(color-11)",
+    CounterBadgeTextColor "color(color-15)",
+  );
+}
```

Comment:
This should also be removed :)
You removed the -st-default which imports the CounterBadge but forgot to remove the usage.

Reply:
I see that you remove all the visual tests for the overrides. So, you can just remove this file.